Expensify/App
# Move config files to a specific config folder

### Fixed Issues
Part of https://github.com/Expensify/Expensify/issues/146267

### Tests
1. Run `npm run web`
1. Verify the app builds
1. Make an eslint error in a file
1. Verify that the error is caught in the CLI
